Configuration/FlexForms/Plugin.xml

```diff
@@ -1,39 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<T3DataStructure>
-    <sheets>
-        <general>
-            <ROOT>
-                <TCEforms>
-                    <!--
-                        The sheet title will not be visible in the BE. Still, the XML element needs to be present to
-                        avoid a crash in the BE.
-                    -->
-                    <sheetTitle/>
-                </TCEforms>
-                <type>array</type>
-                <el>
-                    <switchableControllerActions>
-                        <TCEforms>
-                            <label>LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea.view</label>
-                            <onChange>reload</onChange>
-                            <config>
-                                <type>select</type>
-                                <renderType>selectSingle</renderType>
-                                <items type="array">
-                                    <numIndex index="1" type="array">
-                                        <numIndex index="0">LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea.index</numIndex>
-                                        <numIndex index="1">Tea->index</numIndex>
-                                    </numIndex>
-                                    <numIndex index="2" type="array">
-                                        <numIndex index="0">LLL:EXT:tea/Resources/Private/Language/locallang.xlf:plugin.tea.show</numIndex>
```

Comment:
We also should drop the labels from the XLIFF files or reorganize the labels so that they are used in `Configuration/TCA/Overrides/tt_content.php`.